common/static/common/js/utils/edx.utils.validate.js

```diff
@@ -108,7 +108,7 @@
                             [
                                 '(^[-!#$%&\'*+/=?^_`{}|~0-9A-Z]+(\\.[-!#$%&\'*+/=?^_`{}|~0-9A-Z]+)*',
                                 '|^"([\\001-\\010\\013\\014\\016-\\037!#-\\[\\]-\\177]|\\\\[\\001-\\011\\013\\014\\016-\\177])*"',
-                                ')@((?:[A-Z0-9](?:[A-Z0-9-]{0,61}[A-Z0-9])?\\.)+[A-Z]{2,6}\\.?$)',
+                                ')@((?:[A-Z0-9](?:[A-Z0-9-]{0,61}[A-Z0-9])?\\.)+[A-Z]{2,12}\\.?$)',
```

Comment:
Are we sure that 12 characters will be enough this time?

Reply:
I think that 12 characters will be enough. They want to register emails like that `******@secdev.foundation`

Reply:
Okey, so 🤞